general-javascript-best-practices.md

```diff
@@ -149,6 +149,18 @@ Use instead of underscore, due to correct semantic versioning, better performanc
 
 Please only require the methods you need rather than the whole library in order to keep build sizes at a minimum.
 
+**Good:**
```

Comment:
I agree if it's not possible to use `_.chain` in this way but all other methods can still be required. I'll merge this and then we can add another PR to discuss and advise on `_.chain`